projects/wave-core/src/lib/operators/types/gdal-source-type.model.ts

```diff
@@ -62,12 +62,17 @@ export class GdalSourceType extends OperatorType {
                     return {
                         channel: channelDict.channel,
                         datatype: channelDict.datatype,
-                        file_name: channelDict.file_name,
+                        file_name: channelDict.file_name ? channelDict.file_name : undefined,
+                        path: channelDict.path ? channelDict.path : undefined,
                         unit: Unit.fromDict(channelDict.unit),
+                        netcdf_subdataset: channelDict.netcdf_subdataset ? channelDict.netcdf_subdataset : undefined,
                     };
                 }) : undefined,
+                file_name: dict.gdal_params.file_name ? dict.gdal_params.file_name : undefined,
+                path: dict.gdal_params.path ? dict.gdal_params.path : undefined,
                 coords: dict.gdal_params.coords,
                 provenance: dict.gdal_params.provenance,
+                netcdf_subdataset: dict.gdal_params.netcdf_subdataset ? dict.gdal_params.netcdf_subdataset : undefined,
```

Comment:
same here

Reply:
variable is not required by mapping here